content/07.pathogenesis.md

```diff
@@ -446,17 +445,29 @@ Data and empirically determined biological mechanisms inform models, while model
 Many current efforts to model R~t~ have also led to tools that assist the visualization of estimates in real time or over recent intervals [@url:https://epiforecasts.io/covid; @url:https://rt.live].
 These are valuable resources, yet it is also important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.
 
-### Molecular Signatures and Transmission
+### Molecular Signatures, Transmission, and Variants of Concern
```

Comment:
This is just a suggestion rather than making an entirely new section, also shorten as you see fit